packages/vue-app/template/router.js

```diff
@@ -101,7 +101,7 @@ export const routerOptions = {
   fallback: <%= router.fallback %>
 }
 
-export function createRouter (ssrContext, config) {
+export function createRouter (ssrContext, config, store) {
```

Comment:
It seemed the new parameter is not used anywhere, can you provide more details about the usage of the store parameter?

Reply:
@clarkdo It seems usable for custom integrations with nuxt/router module. Indeed we can simply omit param here as is not used by default nuxt router.
```suggestion
export function createRouter (ssrContext, config) {
```